syspage.c

```diff
@@ -391,23 +391,30 @@ void syspage_showMaps(void)
 		lib_printf("%d%-3s %-8s 0x%08x%4s 0x%08x%4s 0x%08x%4s 0x%08x%4s %s\n", pmap->map.id, "", pmap->map.name,
 			pmap->map.start, "", pmap->map.end, "", pmap->top, "", pmap->map.end - pmap->top, "", attr);
 	}
+
+	hal_memInfo();
```

Comment:
This function is redundant. All information about maps should be displayed by map command belonging to hardware independent part.

Reply:
Removed.